docs/reference/inference/service-amazon-bedrock.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[infer-service-azure-ai-studio-api-path-params]]
+==== {api-path-parms-title}
+
+`<inference_id>`::
+(Required, string)
+include::inference-shared.asciidoc[tag=inference-id]
+
+`<task_type>`::
+(Required, string)
+include::inference-shared.asciidoc[tag=task-type]
++
+--
+Available task types:
+
+* `completion`,
+* `text_embedding`.
+--
+
+[discrete]
+[[infer-service-amazon-bedrock-api-request-body]]
+==== {api-request-body-title}
+
+`service`::
+(Required, string) The type of service supported for the specified task type.
+In this case,
+`amazonbedrock`.
+
+`service_settings`::
+(Required, object)
+include::inference-shared.asciidoc[tag=service-settings]
++
+--
+These settings are specific to the `amazonbedrock` service.
+--
+
+`access_key`:::
+(Required, string)
+A valid AWS access key that has permissions to use Amazon Bedrock and access to models for inference requests.
+
+`secret_key`:::
+(Required, string)
+A valid AWS secret key that is paired with the `access_key`.
+To create or manage access and secret keys, see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing access keys for IAM users] in the AWS documentation.
+
+IMPORTANT: You need to provide the access and secret keys only once, during the {infer} model creation.
+The <<get-inference-api>> does not retrieve your access or secret keys.
+After creating the {infer} model, you cannot change the associated key pairs.
+If you want to use a different access and secret key pair, delete the {infer} model and recreate it with the same name and the updated keys.
+
+`provider`:::
+(Required, string)
+The model provider for your deployment.
+Note that some providers may support only certain task types.
+Supported providers include:
+
+* `amazontitan` - available for `text_embedding` and `completion` task types
+* `anthropic` - available for `completion` task type only
+* `ai21labs` - available for `completion` task type only
+* `cohere` - available for `text_embedding` and `completion` task types
+* `meta` - available for `completion` task type only
+* `mistral` - available for `completion` task type only
+
+`model`:::
+(Required, string)
+The base model ID or an ARN to a custom model based on a foundational model.
+The base model IDs can be found in the https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html[Amazon Bedrock model IDs] documentation.
+Note that the model ID must be available for the provider chosen, and your IAM user must have access to the model.
+
+`region`:::
+(Required, string)
+The region that your model or ARN is deployed in.
+The list of available regions per model can be found in the https://docs.aws.amazon.com/bedrock/latest/userguide/models-regions.html[Model support by AWS region] documentation.
+
+`rate_limit`:::
+(Optional, object)
+By default, the `azureaistudio` service sets the number of requests allowed per minute to `240`.
+This helps to minimize the number of rate limit errors returned from Azure AI Studio.
```

Comment:
```suggestion
This helps to minimize the number of rate limit errors returned from Amazon Bedrock.
```

Reply:
Argh - great catches ;) That's what I get for copy / pasting